go/vt/vtgate/safe_session.go

```diff
@@ -543,13 +542,31 @@ func (session *SafeSession) SetReservedConn(reservedConn bool) {
 func (session *SafeSession) SetPreQueries() []string {
```

Comment:
don't we want to change the return type to be a single string instead of a slice?

Reply:
currently the vttablet api takes in []string so we cannot change that one release.